cmd/aro/operator.go

```diff
@@ -77,6 +93,11 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 			kubernetescli)).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to create controller AlertWebhook: %v", err)
 		}
+		if err = (workaround.NewReconciler(
+			log.WithField("controller", controllers.WorkaroundControllerName),
+			kubernetescli, configcli, mcocli, clustercli, arocli, restConfig)).SetupWithManager(mgr); err != nil {
+			return fmt.Errorf("unable to create controller Workaround: %v", err)
```

Comment:
Is the workaround controller generic for all workarounds or should we use a more specific name here?

Reply:
yeah, it's generic. So Troy can use this as well